pip_review/__main__.py

```diff
@@ -83,6 +83,9 @@ def parse_args():
     parser.add_argument(
         '--continue-on-fail', '-C', action='store_true', default=False,
         help='Continue with other installs when one fails')
+    parser.add_argument(
+        '--freeze-outdated-packages', '-f', action='store_true', default=False,
```

Comment:
Please remove `-f` as a short option, because it conflicts with pip's `--find-links` option. If you have a strong desire for a short option, you can use the capital `-F`.

Reply:
Thanks for pointing out, fixed!